### ISSUE TITLE
Add favicon

### ISSUE THREAD

**Issue by 2opremio** [2opremio/ssweb#4]
We don't have a favicon, we could create one based on this image:

![squarelogo](https://user-images.githubusercontent.com/2362916/39136487-a2e4212e-471b-11e8-85d6-b4fa74d4858f.png)

**Comment by 2opremio** [comment]
I used https://www.favicon-generator.org/ , which generated the following:

> Download the generated favicon and extract the contents to the root directory of your site:

[e63acf95dbf31da05d2545eacf959a24.ico.zip](https://github.com/2opremio/ssweb/files/1938968/e63acf95dbf31da05d2545eacf959a24.ico.zip)

> Include the following code in the head of your HTML document.
```html
<link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png">
<link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png">
<link rel="apple-touch-icon" sizes="72x72" href="/apple-icon-72x72.png">
<link rel="apple-touch-icon" sizes="76x76" href="/apple-icon-76x76.png">
<link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png">
<link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png">
<link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png">
<link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
<link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
<link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/manifest.json">
<meta name="msapplication-TileColor" content="#ffffff">
<meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
<meta name="theme-color" content="#ffffff">
```

@wanecek is all the above necessary? Seems cumbersome just to get a favicon. Happy to do it if you think it's necessary

**Comment by wanecek** [comment]
Fixed in a760d0d87edb1d80af05815fbd014573cebc7690 :)
